ext/pdo_sqlite/pdo_sqlite.stub.php

```diff
@@ -33,6 +33,16 @@ class Sqlite extends \PDO
     /** @cvalue PDO_SQLITE_ATTR_EXTENDED_RESULT_CODES */
     public const int ATTR_EXTENDED_RESULT_CODES = UNKNOWN;
 
+    /** @cvalue SQLITE_OK */
+    public const int OK = UNKNOWN;
```

Comment:
These could be enums in theory for the authorizer callback, although OK is more generally applicable too